docs-new/docs/contribute/software/dev-environ.md

```diff
@@ -144,28 +144,51 @@ Tests are run on your local machine.
 Study the detailed log that is produced to identify why the test failed.
 Study these errors, modify your code, and rerun the test until it passes.
 
-1. Use your IDE to run unit tests on your code.
+### Unit tests
 
-1. Run the integration tests from the root directory of your clone:
+Use your IDE to run unit tests on your code.
 
-```shell
-make integration-test-local
-```
+### Integration tests
 
-   `integration-test-local` cleans up after the test.
+Run the integration tests from the root directory of your clone:
 
-1. From the `lifecycle-operator` directory, run the component test:
+  ```shell
+  make integration-test-local
+  ```
 
-```shell
-make component-test
-```
+`integration-test-local` cleans up after the test.
+
+### Component test
+
+From the `lifecycle-operator` directory, run the component test:
 
-1. From the `lifecycle-operator` directory, run the end-to-end tests:
+  ```shell
+  make component-test
+  ```
+
+### End-to-end test
+
+From the `lifecycle-operator` directory, run the end-to-end tests:
+
+  ```shell
+  make e2e-test
+  ```
+
+### Kuttl test
+
+Run the Kuttl [KUbernetes Test TooL](https://kuttl.dev/)
+if your PR modifies one of the operators.
+The syntax is:
 
 ```shell
-make e2e-test
+kubectl kuttl test --start-kind=false ./test/integration/ \
+        --config=kuttl-test.yaml --test name-of-your-test-directory
```

Comment:
this would work only for the integration folder now we do have multiple folders each containing some test I agree with Mo it would be easier to point to the [make file here ](https://github.com/keptn/lifecycle-toolkit/blob/main/Makefile#L25-L53)

Reply:
we should also probably rename integration folder to lifecycle 😄

Reply:
Rewritten to show kuttl test info under integration tests